plugin-server/src/utils/db/db.ts

```diff
@@ -1099,12 +1098,12 @@ export class DB {
         return queryResult.data as PluginLogEntry[]
     }
 
-    public async queuePluginLogEntry(entry: LogEntryPayload): Promise<void> {
+    public queuePluginLogEntry(entry: LogEntryPayload): Promise<void> {
```

Comment:
nitpick: Can we skip the return value altogether? We don't await anything and we never reject the returned promise.

Reply:
Not really. the reason I'm not changing it is that this would require a bunch more type changes in plugin code that I really don't want to do (remember this PR is me trying to a smaller chunk rather than all at once).